### PR TITLE
PROJ-181 Better support links and billing address

### PR DESCRIPTION
**Pull request by LucWollants**
### Fixed
- Fixed getting links from a project (either contact or organisation)
- Fixed reading the billing address

Note: this requires a better solution as proposed in https://jira.uitdatabank.be/browse/PROJ-156

---
Ticket: https://jira.uitdatabank.be/browse/PROJ-181

### CODE REVIEW

**Review by jdrieghe — APPROVED** [review]

**Review by jdrieghe — APPROVED** [review]
I would've kept those annotation cleanups for a separate PR though 🙈

File: src/Insightly/InsightlyClient.php [hunk 1]
```diff
@@ -304,4 +305,12 @@ public function updateOrganisation(Organisation $organisation)
     {
         return GetOrganisationResult::parseToResult($this->request(RequestInterface::PUT, 'Organisations/', null, json_encode($organisation->toInsightly())));
     }
+
+    /**
+     * {@inheritdoc}
+     */
```
Comment (bertramakers) [inline src/Insightly/InsightlyClient.php]: I noticed you added an `@inheritdoc` annotation here, probably to stay consistent with the others in this file. I've gone ahead and just removed all `@inheritdoc` annotations

**Review by bertramakers — APPROVED** [review]

**Comment by bertramakers** [comment]
> I would've kept those annotation cleanups for a separate PR though 🙈

Hmm I can cherry-pick them to a different branch and remove them here if you really want. I was just going to remove the `@inheritdoc` added in this PR and merge it, but then noticed it was easy to just remove all of them.

**Comment by jdrieghe** [comment]
Wouldn't bother anymore right now. Feel free to merge.

File: src/Insightly/InsightlyClient.php (hunk 1 above)
Comment (LucWollants) [reply src/Insightly/InsightlyClient.php]: Yeah, I did not wanted to go down the rabbit hole of updating code styling until the PHP-CS-Fixer was added.

**Pull request merged**